src/nanoserver/1809/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1809
```

Comment:
In case I haven't mentioned this before, I generally recommend changing the escape character in Windows Dockerfiles.  There are scenarios when the default will cause you aggravation - see https://docs.docker.com/engine/reference/builder/#escape.  It may save you a headache down the road.

Reply:
Updated.